cmd/pipehub/main.go

```diff
@@ -17,9 +18,16 @@ import (
 
 var done = make(chan os.Signal, 1)
 
+var (
+	gitCommit string
+	goVersion string
```

Comment:
Here you could do something different: `runtime.Version()`.

Reply:
good idea